samples/testing-frameworks/appium/client-side/csharp/biometric-sample-test/BitbarBiometricSample/TestBiometricsAndroid.cs

```diff
@@ -0,0 +1,79 @@
+using OpenQA.Selenium.Appium.Android;
+
+namespace BitbarBiometricSample
+{
+    [TestFixture]
+    public class TestBiometricsAndroid
+    {
+        private readonly String SCREENSHOT_DIRECTORY = AppDomain.CurrentDomain.BaseDirectory + "/screenshots/";
+        private RemoteWebDriver driver;
+
+        [OneTimeSetUp]
+        public void InitDriver()
+        {
+            const String AppiumHubURL = "https://appium.bitbar.com/wd/hub";
+
+            AppiumOptions capabilities = new();
+            //Platform dependent
+            capabilities.AddAdditionalCapability("platformName", "Android");
+            capabilities.AddAdditionalCapability("deviceName", "Android Phone");
+            capabilities.AddAdditionalCapability("automationName", "Appium");
+            //Customizable
+            capabilities.AddAdditionalCapability("bitbar_project", "C# Appium Automated Test");
+            capabilities.AddAdditionalCapability("bitbar_testrun", "Android Run");
+            capabilities.AddAdditionalCapability("bitbar_device", "Google Pixel 2");
+            //Required
+            capabilities.AddAdditionalCapability("bitbar_apiKey", "<YOUR_BITBAR_API_KEY>");
```

Comment:
What do you think about extracting caps to another file? Especially bitbar_apiKey and bitbar_app